src/ir/constant-fp.cc

```diff
@@ -51,6 +51,17 @@ NAN_METHOD(ConstantFPWrapper::getNaN) {
     info.GetReturnValue().Set(ConstantWrapper::of(nan));
 }
 
+NAN_METHOD(ConstantFPWrapper::getInfinity) {
+    if (info.Length() != 1 || !TypeWrapper::isInstance(info[0])) {
```

Comment:
Could you please add support for the optional negative case as well [getInfity](http://llvm.org/doxygen/classllvm_1_1ConstantFP.html#a6f2864671b22c9f6135b6ae5d41bcf54)?

Reply:
Yes, sure!